python/ribasim/tests/test_model.py

```diff
@@ -143,3 +145,20 @@ def test_write_adds_fid_in_tables(basic, tmp_path):
         assert "fid" in df.columns
         fids = df.get("fid")
         assert fids.equals(pd.Series(range(1, len(fids) + 1)))
+
+
+def test_model_merging(basic, subnetwork, tmp_path):
+    model = deepcopy(basic)
+    model_added = deepcopy(subnetwork)
+    model.merge_model(model_added)
+    model.merge_model(model_added)
```

Comment:
Why twice?

Reply:
The added model is a subnetwork with `allocation_network_id = 1` (which will probably be changed in the future because that is reserved for the main network), so I check with `assert model.max_allocation_network_id() == 2` whether a shift was added to the `allocation_network_id` of the second one. I added some clarification in the code. I'll also test the coordinate translation while I'm at it.